cloup/_commands.py

```diff
@@ -164,7 +162,7 @@ def add_command(
     ) -> None:
         super().add_command(cmd, name, section, fallback_to_default_section)
         name = cast(str, cmd.name) if name is None else name
-        aliases = getattr(cmd, 'aliases', [])
+        aliases = getattr(cmd, 'aliases', None) or ()
```

Comment:
I don't think this change is needed. `self.aliases` is always a `List[str]` (eventually empty).

Reply:
Yeah, I thought you might say this. :-) In my [fork of *typer*](https://github.com/alexreg/typer), I was allowing it to be a tuple, and this made things easier... but of course I don't blame you if you only want to allow lists.